test_scripts/Defects/8_1/980_SDL_responds_INVALID_DATA_in_case_soft_button_has_Type_Image_or_Both_and_Text_has_specific_string.lua

```diff
@@ -22,38 +22,128 @@ local utils = require('user_modules/utils')
+    softButtons = {
+      {
+        type = "BOTH",
+        text = "Close\t",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_BOTH_Text_lineBreak = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "BOTH", text = "Close\n", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" }},
+    softButtons = {
+      {
+        type = "BOTH",
+        text = "Close\n",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_BOTH_Text_empty = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "BOTH", text = "", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" }},
+    softButtons = {
+      {
+        type = "BOTH",
+        text = "",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_IMAGE_Text_whitespace = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "IMAGE", text = "        ", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" }},
+    softButtons = {
+      {
+        type = "IMAGE",
+        text = "        ",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_IMAGE_Text_HorizontalTab = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "IMAGE", text = "Close\t", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" } },
+    softButtons = {
+      {
+        type = "IMAGE",
+        text = "Close\t",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_IMAGE_Text_lineBreak = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "IMAGE", text = "Close\n", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" }},
+    softButtons = {
+      {
+        type = "IMAGE",
+        text = "Close\n",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
+      }
+    }
   },
   type_IMAGE_Text_empty = {
-    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
-    softButtons = { { type = "IMAGE", text = "", softButtonID = 821, } }
+    ttsChunks = {{ text = "FirstAlert", type = "TEXT" }},
+    softButtons = {
+      {
+        type = "IMAGE",
+        text = "",
+        image = {
+          value = "icon.png",
+          imageType = "DYNAMIC",
+        },
+        softButtonID = 821
```

Comment:
Since the code below is the same for all the cases please move it in a separate variable.
```
        image = {
          value = "icon.png",
          imageType = "DYNAMIC",
        },
        softButtonID = 821
```